### ISSUE TITLE
Google's Datasource Error

### ISSUE THREAD

**Issue by efbe2121** [hystax/optscale#80]
Error message : 403 GET https://compute.googleapis.com/compute/v1/projects/**projectname**/regions: Required 'compute.regions.list' permission for 'projects/**projectname**'

Where ? 
1. When connecting Google Datasource
2. When restarting restapi-pod, the connecting data source went well, but after several moment, the import will failed **except for Bucket**

Things tried : (no luck)
1. Giving Owner permission to Service Account
2. Full permission to the Role
```
bigquery.jobs.create
bigquery.jobs.get
bigquery.jobs.list
bigquery.jobs.update
bigquery.tables.get
bigquery.tables.getData
bigquery.tables.list
compute.addresses.get
compute.addresses.list
compute.addresses.setLabels
compute.disks.get
compute.disks.list
compute.disks.setLabels
compute.globalAddresses.get
compute.globalAddresses.list
compute.globalAddresses.setLabels
compute.instances.get
compute.instances.list
compute.instances.setLabels
compute.machineTypes.get
compute.machineTypes.list
compute.networks.get
compute.networks.list
compute.regions.get
compute.regions.list
compute.snapshots.get
compute.snapshots.list
compute.snapshots.setLabels
compute.zones.get
compute.zones.list
monitoring.timeSeries.list
storage.buckets.get
storage.buckets.list
storage.buckets.update
```

How to produce? 

`Install latest version opensource, configure datasource, configure billing export, then connect`

**Comment by efbe2121** [comment]
Please any update on this?

**Comment by maxb-hystax** [comment]
Hi Kristian!
We're checking it on our side, will let you know.

**Comment by efbe2121** [comment]
> Hi Kristian! We're checking it on our side, will let you know.

Also, please check out the Recommendation Section aswell for the GCP Source. I highly doubt that there is no recommendation for me, since our usage is pretty high

![image](https://github.com/hystax/optscale/assets/44221453/ed8dbce0-0ddf-40e6-b31e-21484a366332)

The only recommendation i get from google is the volume, not the other aspect

**Comment by efbe2121** [comment]
@maxb-hystax  Any update on this? I've tried the newest release 2001-public
I cant even add any Google's datasources.

Checked the console :  
![image](https://github.com/hystax/optscale/assets/21376944/4a714330-ec60-47aa-af94-ca5b91cae986)
Error message : Cloud connection error: Failed to connect to the cloud with provided credentials - Attempted to set connect timeout to 0, but the timeout cannot be set to a value less than or equal to 0.

Followed the docs, or is there any updated docs for google?

**Comment by maxb-hystax** [comment]
Hi!
Just delivered update with the possible fix - https://github.com/hystax/optscale/commit/22b5d7a3cd48641b19db964a2400ca9c679db490
Please try and let us know the result.

**Comment by efbe2121** [comment]
> Hi! Just delivered update with the possible fix - [22b5d7a](https://github.com/hystax/optscale/commit/22b5d7a3cd48641b19db964a2400ca9c679db490) Please try and let us know the result.

Can you please help me how to update?
I try to rebuild the RestAPI after changing the values from commit you mentioned, but seems nothing changed
Eventho the values on setup.py in the running container already changed

On restapi container i tried "pip list"
![image](https://github.com/hystax/optscale/assets/44221453/230d42e8-5a6d-4150-8744-c3680e8ce01f)

**Comment by efbe2121** [comment]
Nevermind, i did every other images that depends on the tools/cloud_adapter.
